src/iap/shared/services/api.service.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { EnvironmentService } from './environment.service';
+
+@Injectable({ providedIn: 'root' })
+export class ApiService {
+	api!: string;
+	constructor(private http: HttpClient, private environmentService: EnvironmentService) {
+		this.api = this.environmentService.getValue('api');
+	}
+
+	get(path: string) {
+		return this.http.get(`${this.api}${path}`);
+	}
```

Comment:
Don't you want to add generics to these methods? Example:
```suggestion
	get<T>(path: string) {
		return this.http.get<T>(`${this.api}${path}`);
	}
```